### PR TITLE
Fix registering account with same email

### PR DESCRIPTION
**Pull request by pallix**

### CODE REVIEW

**Comment by pallix** [comment]
maybe the Colander preparer  should be moved to the `Email` schema itself.

File: src/adhocracy_core/adhocracy_core/evolution/__init__.py [hunk 1]
```diff
@@ -213,6 +214,19 @@ def hide_password_resets(root):  # pragma: no cover
...
+def lower_case_users_emails(root):  # pragma: no cover
+    """Lower case users email."""
+    users = find_service(root, 'principals', 'users')
+    for user in users.values():
+        if not IUserExtended.providedBy(user):
+            return
+        sheet = get_sheet(user, IUserExtended)
+        sheet.set({'email': user.email.lower()})
+        # force update. Not working otherwise even if the sheet is set?!
+        user._p_changed = True
```
Comment (joka) [inline src/adhocracy_core/adhocracy_core/evolution/__init__.py]: This is a serious bug, i will have a look at it

**Comment by joka** [comment]
> maybe the Colander preparer should be moved to the Email schema itself.

yup, but for its also fine that way

**Comment by joka** [comment]
lgtm, can you merge master?

**Comment by pallix** [comment]
Master merged

**Pull request merged**